tests/observability/scripts/access_log.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "${REPO_ROOT}/content/en/docs/tasks/observability/logs/access-log/snips.sh"
+source "${REPO_ROOT}/tests/util/samples.sh"
+
+
+
+
+kubectl label namespace default istio-injection=enabled --overwrite
+startup_sleep_sample # needed for sending test requests with curl
+startup_httpbin_sample
+_run_and_verify_contains  snip_test_the_access_log_1 "server: envoy"
```

Comment:
_run_and_verify_contains is renamed to _verify_contains in https://github.com/istio/istio.io/pull/7452
```suggestion
_verify_contains  snip_test_the_access_log_1 "server: envoy"
```